src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -1,59 +1,56 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{80721425-68E5-48DC-87EA-41D63D0B45FA}</ProjectGuid>
+    <ProjectGuid>{74165807-676F-4464-B607-8770C13E8991}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>FakeItEasy</RootNamespace>
     <AssemblyName>FakeItEasy</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
I'd prefer not to have the StyleCop warnings, even if it means more
intricate #if's
On 13 Jul 2016 23:25, "Jeremy Meng" <notifications@github.com> wrote:

> In src/FakeItEasy/FakeItEasy.csproj
> <https://github.com/FakeItEasy/FakeItEasy/pull/617#discussion_r70711833>:
>
> >      <OutputType>Library</OutputType>
> >      <AppDesignerFolder>Properties</AppDesignerFolder>
> >      <RootNamespace>FakeItEasy</RootNamespace>
> >      <AssemblyName>FakeItEasy</AssemblyName>
> > -    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
> > +    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
>
> Are we fine with getting @thomaslevesque
> <https://github.com/thomaslevesque>'s changes in with the SA warnings?
> jeremymeng#1 <https://github.com/jeremymeng/FakeItEasy/pull/1>
>
> I forgot whether I have asked it or not: why is the test project targeting
> v4.5 instead of v4.0?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/617/files/af0b1b76c0ebfc59c07fc79859885f8bef22d964#r70711833>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AApXSL1sCP2DSH-oKTDPtqzv7WsnMQ8Bks5qVVfLgaJpZM4HjNMh>
> .
>